scripts/linters/pylint_extensions_test.py

```diff
@@ -2590,3 +2590,150 @@ def test_file_overview_at_end_of_file(self):
+            temp_file.close()
+
+    def test_inline_comment_after_class_docstring(self):
+        node_inline_comment_after_class_docstring = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(dummy_class):
+                        \"\"\"This is a docstring.\"\"\"
+                        # This is a comment.
+                        def func():
+                            a = 1 + 2
+                """)
+        node_inline_comment_after_class_docstring.file = filename
+        node_inline_comment_after_class_docstring.path = filename
+
+        self.checker_test_object.checker.visit_classdef(
+            node_inline_comment_after_class_docstring)
+
+        message = testutils.Message(
+            msg_id='newline-below-class-docstring',
+            node=node_inline_comment_after_class_docstring)
+
+        with self.checker_test_object.assertAddsMessages(message):
+            temp_file.close()
+
+    def test_class_argument_in_other_lines(self):
+        node_class_argument_in_other_lines = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(
+                            dummy_class):
+                        \"\"\"This is a docstring.\"\"\"
+
+                        a = 1 + 2
+                """)
+        node_class_argument_in_other_lines.file = filename
+        node_class_argument_in_other_lines.path = filename
+
+        self.checker_test_object.checker.visit_classdef(
+            node_class_argument_in_other_lines)
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
+
+    def test_single_newline_below_class_docstring(self):
+        node_with_no_error_message = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(dummy_class):
+                        \"\"\"This is a multiline docstring.\"\"\"
+
+                        a = 1 + 2
+                """)
+        node_with_no_error_message.file = filename
+        node_with_no_error_message.path = filename
+
+        self.checker_test_object.checker.visit_classdef(
+            node_with_no_error_message)
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
```

Comment:
Add test for class with:
!. No doc
2. A newline above docstring and newline after doc string
3.  A newline above docstring and no newline after doc string

Reply:
Done.